test/tsconfig.json

```diff
@@ -2,6 +2,10 @@
   "extends": "@salesforce/dev-config/tsconfig-test-strict-esm",
   "include": ["./**/*.ts"],
   "compilerOptions": {
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "baseUrl": "..",
+    "paths": {
```

Comment:
are these change still needed? same with `./tsconfig`

Reply:
No, only for the dev release.  I'll revert.